www/src/routes/Landing.tsx

```diff
@@ -0,0 +1,214 @@
+// Libraries
+import React, { FC } from 'react';
+
+// Material UI
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Box, makeStyles } from '@material-ui/core';
+
+// Assets
+import PairProgramming from '../assets/pair_programming.svg';
+import LandingBackground from '../assets/LandingBackground.svg';
+import BlackLogo from '../assets/logo_black.svg';
+import Practice from '../assets/practice.svg';
+import MockInterview from '../assets/mock_interview.svg';
+import Industry from '../assets/industry.svg';
+import ResumeReview from '../assets/resume_review.svg';
+import Teamwork from '../assets/teamwork.svg';
+import Support from '../assets/support.svg';
+
+// Components
+import OutlinedButton from '../components/OutlinedButton';
+
+export const Landing: FC = () => {
+    return (
+        <Grid container direction='row' justify='center' style={{ backgroundImage: `url(${LandingBackground})`, backgroundSize: 'cover', paddingBottom: 80 }}>
+            <Grid container item spacing={10} xs={10}>
+                <Intro />
+                <Values />
+                <Services />
+            </Grid>
+        </Grid>
+    );
+};
+
+const Intro: FC = () => {
+    const classes = useStyles();
+
+    return (
+        <Grid container item>
```

Comment:
I know nothing about Material UI. @armiantos are you happy with using Grids for layouts like this. AFAICT seems fine.